deployment/live/gcp/static-ct-staging/logs/arche2025h1/README.md

```diff
@@ -0,0 +1,62 @@
+# Arche2025h1 staging log
+
+## Overview
+
+This config uses the [gcp/tesseract/conformance](/deployment/modules/gcp/tesseract/conformance) module to
+define a Tesseract log Cloud Run, backed by Trillian Tessera and preloaded with argon2025h1 entries.
+
+### Update the roots
+Run the following command from the root of the repository.
+
+It fetches roots from argon2025h1, and filters out roots that don't parse with `crypto/x509` and `internal/lax509`.
+As of 2025-03-21, these roots are:
+- https://crt.sh/?id=298, with a negative serial number
+- https://crt.sh/?id=9027356, affected by https://github.com/golang/go/issues/69463.
+
+```
+go run github.com/google/certificate-transparency-go/client/ctclient@master get-roots --log_uri=https://ct.googleapis.com/logs/us1/argon2025h1/ --text=false | \
+awk \
+  '/-----BEGIN CERTIFICATE-----/{c=1; pem=$0; show=1; next}
+   c{pem=pem ORS $0}
+   /-----END CERTIFICATE-----/{c=0; if(show) print pem}
+   ($0=="MIIFxDCCBKygAwIBAgIBAzANBgkqhkiG9w0BAQUFADCCAUsxGDAWBgNVBC0DDwBT"||$0=="MIIFVjCCBD6gAwIBAgIQ7is969Qh3hSoYqwE893EATANBgkqhkiG9w0BAQUFADCB"){show=0}' \
+   > deployment/live/gcp/static-ct-staging/logs/arche2025h1/roots.pem
+```
+
+
+### Automatic Deployment
+This GCP SCTFE preloaded staging environment is designed to be deployed by the Cloud Build ([Terraform module](/deployment/modules/gcp/cloudbuild/preloaded/), [Terragrunt configuration](/deployment/live/gcp/static-ct-staging/cloudbuild/staging/)).
+
+### Manual Deployment
+TODO(phboneff): come back to this, CloudRun doesn't trigger a deployment if the tag value change value.
```

Comment:
```suggestion
TODO(phboneff): come back to this, Cloud Run doesn't trigger a deployment if the tag value change value.
```

Reply:
Do you mean the tag is re-tagged to another commit and force pushed?

Reply:
I couldn't parse my own sentence, sorry. If a Cloud Run service was started via terraform with a container tagged with "latest", a new (different!) container is tagged "latest", and the terraform config is applied again, the Cloud Run service won't restart. A fix is to use tags that are non stable across containers, such as their SHORT_HASH. It's not an issue for the ci stack, because we bring it down and up every time.